spec/services/booking_request_creator_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe BookingRequestCreator do
+  let(:prisoner_step) {
+    PrisonerStep.new(
+      prison_id: 1,
+      first_name: 'Oscar',
+      last_name: 'Wilde',
+      date_of_birth: Date.new(1980, 12, 31),
+      number: 'a1234bc'
+    )
+  }
+
+  let(:visitors_step) {
+    VisitorsStep.new(
+      first_name: 'Ada',
+      last_name: 'Lovelace',
+      date_of_birth: Date.new(1970, 11, 30),
+      email_address: 'ada@test.example.com',
+      phone_no: '01154960222'
+    )
+  }
+
```

Comment:
My feeling is that it wouldn't be useful, because it's the values that are important and that are being checked below. If we had factories, we could change `VisitorsStep.new(` to `build(:visitors_step, `, but they'd otherwise be much the same.